.github/workflows/lint_test_charts.yaml

```diff
@@ -83,6 +83,11 @@ jobs:
       - name: Create test ns
         run: kubectl create ns ct
 
+      - name: Test Flux CD on the kind instance that is created by this CI/CD
```

Comment:
If this is not for debugging purposes I'd either remove it or add a comment explaining why it is needed and for which chart if we cannot test it just for the SA one

Reply:
This is the commit that introduced this. It is a test.

I was unable to know why the CI was failing tests because the chart testing action does not give you logs of failing pre-install hooks.

![Screenshot 2023-10-25 at 10 30 31](https://github.com/newrelic/helm-charts/assets/53659978/628ab387-8436-48d5-875f-b2e7a8065a07)